app/workers/start_of_cycle_notification_worker.rb

```diff
@@ -0,0 +1,73 @@
+class StartOfCycleNotificationWorker
+  include Sidekiq::Worker
+
+  def perform(service, batch_size = 100)
+    @service = service
+
+    providers_scope.find_each(batch_size: batch_size) do |provider|
+      provider.provider_users.where.not(Arel.sql(chaser_sent_sql)).each do |provider_user|
+        ProviderMailer.send(mailer_method, provider_user)
+        ChaserSent.create!(chased: provider_user, chaser_type: mailer_method)
+
+        next if service == :apply
+        next unless provider_user.provider_permissions.find_by(provider: provider).manage_organisations
+        next if ProviderSetup.new(provider_user: provider_user).relationships_pending.blank?
+        next if ChaserSent.exists?(chased: provider_user, chaser_type: setup_mailer_method)
```

Comment:
do we need to check this again here? 🤔

Reply:
I don't think it serves any purpose no, I'll remove it, if we error before sending the permissions email but after sending the general email it will skip the user anyway, which might be an issue in itself.

Reply:
see new suggestion: https://github.com/DFE-Digital/apply-for-teacher-training/pull/5593#pullrequestreview-763918784